examples/src/main/java/org/apache/ignite/examples/ml/naivebayes/CompoundNaiveBayesExample.java

```diff
@@ -0,0 +1,78 @@
+package org.apache.ignite.examples.ml.naivebayes;
```

Comment:
Could you please add Apache header containing the license?

Reply:
I added Apache header in all Compound Bayes tests. Sorry for this oversight